ui/app/src/components/Header.tsx

```diff
@@ -147,6 +147,11 @@ export default function Header(): JSX.Element {
       <Toolbar
         sx={{
           backgroundColor: (theme) => theme.palette.designSystem.blue[700],
+          '&': {
+            minHeight: '40px',
+            paddingLeft: 0,
+            paddingRight: 0.75,
```

Comment:
```suggestion
            paddingRight: 2,
```
In order to have the same padding as "body" content

Reply:
Yeah ok it's a matter of either having the button zone aligned with the rest (padding 2)
![image](https://github.com/perses/perses/assets/7058693/3f3f97d1-f05b-496a-a7de-1fa8278b9cc0)
or with the slider itself (padding 0.75)
![image](https://github.com/perses/perses/assets/7058693/fb5c0ebf-2cd9-49f5-a22f-4333120cd6d0)

Personally I have a slight preference for the 2nd one even it looks a bit like a hack here I agree :D.

Reply:
This hover effect is pretty annoying, ok for me